docker-build.sh

```diff
@@ -1,13 +1,28 @@
 #!/bin/bash
```

Comment:
Maybe we should rename this file `docker-build-centos.sh`.

Reply:
Yeah, I had the same thoughts, but was lazy :)